examples/updatecli.generic/helmChart/SCM.2.yaml

```diff
@@ -0,0 +1,40 @@
+title: Bump Jenkins controller docker image tag
+scms:
+  olblak-chart:
+    kind: github
+    spec:
+      user: "updatecli"
+      email: "updatecli2@olblak.com"
+      owner: "olblak"
+      repository: "charts"
+      token: {{ requiredEnv .github.token }}
+      username: "olblak"
+      branch: "master"
+pullrequests:
+  chart:
+    kind: github
+    scmid: olblak-chart
+    targets:
+      - chartjenkins1 
+    spec:
+      labels:
+        - bug
+sources:
+  jenkins:
+    kind: jenkins
+    name: Get Latest Jenkins Stable version
+    spec:
+      release: stable
+      github:
+        token: {{ requiredEnv .github.token }}
+        username: {{ .github.username }}
+targets:
+  chartjenkins1:
+    name: Bump Jenkins controller docker image tag
+    kind: helmChart
+    scmID: olblak-chart
```

Comment:
There are two versions "scmid" & "scmID"?

Reply:
Thank you for pointing this, I am pretty sure that the way we load yaml file, it doesn't care about lowercase/uppercase. so ScMiD could be fine as well.
But that's a good suggestion that I should try to stay consistent

Reply:
Nope, it doesn't work and throws an error if you don't use the exact keyword `scmid`. I thought that the Yaml parser would be case unsensitive but it is: https://github.com/go-yaml/yaml/issues/123 (as per the documentation https://pkg.go.dev/gopkg.in/yaml.v3#Marshal : `Struct fields are only <...> unmarshalled using the field name lowercased as the default key`)

It means that we should use the same everywhere